consumer/pdata/trace.go

```diff
@@ -78,6 +78,34 @@ func (td Traces) SpanCount() int {
 	return spanCount
 }
 
+// SpanCountByResourceStringAttribute calculates the number of spans by resource specified by
+// provided string attribute attrKey.
+func (td Traces) SpanCountByResourceStringAttribute(attrKey string) map[string]int {
```

Comment:
I would not put this in the main API and expose publicly yet. If there are other places where we use this we can decide to move later.

Reply:
Makes sense

Reply:
Moved to the package where it's being used and made it internal there